Makefile

```diff
@@ -0,0 +1,20 @@
+VERSION := $(shell git tag | grep ^v | sort -V | tail -n 1)
+GIT_HASH := $(shell git rev-parse --short HEAD )
+LDFLAGS = -ldflags "-X main.Version=${VERSION} -X main.GitHash=${GIT_HASH}"
+
```

Comment:
Please add .PHONY for the pseudo-targets (https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)

Reply:
k, will add this. This is my first Makefile so the learning curve is pretty steep.